test_vm/vagrant_config_defaults.rb

```diff
@@ -2,6 +2,6 @@
   {nfs: true, nfs_version: 4, mount_options: ["noatime", "nodiratime"]}
 
 # Default options
-CONFIG['memory'] = "1100"
+CONFIG['memory'] = "2048"
```

Comment:
Do we need to change the default? Memory size can be customized in `vagrant_config_sample` (that's needed e.g. for Oracle too).